plugins/rtmp-services/data/services.json

```diff
@@ -3461,6 +3461,9 @@
             "common": false,
             "more_info_link": "https://docs.dolby.io/streaming-apis/docs/using-obs",
             "stream_key_link": "https://streaming.dolby.io",
+            "multitrack_video_configuration_url": "https://director.millicast.com/api/multitrackvideo/configuration",
+            "multitrack_video_name": "Enhanced Broadcasting",
+            "multitrack_video_learn_more_link": "https://docs.dolby.io/streaming-apis/docs/obs-enhanced-broadcasting-multitrack-video",
```

Comment:
The "Learn More" link redirects to a different page that does not mention "Enhanced Streaming" or "Enhanced Broadcasting". My understanding from previous comments was that the docs were published, and we expect them to be available before we merge. Where can the docs for this be found?